### PR TITLE
Add cart management page

### PR DESCRIPTION
**Pull request by darshandamre**
Preview [https://e-com-git-cart-page-darshandamre.vercel.app/pages/cart.html](https://e-com-git-cart-page-darshandamre.vercel.app/pages/cart.html)

### CODE REVIEW

**Review by chanchal-panpaliya — APPROVED** [review]
code looks good.

File: pages/cart.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                1
+                <i class="px-1 fa-solid fa-circle-plus"></i>
+              </p>
+              <button class="btn">Remove from Cart</button>
+              <button class="btn card__btn--outline">Move to wishlist</button>
+            </div>
+          </div>
+
+          <div class="cart__details p-6">
+            <p class="cart__details__title fw-600">PRICE DETAILS</p>
```
Comment (HarshnaNHaswani) [inline pages/cart.html:62]: you can use text-transfom: uppercase for this

**Review by HarshnaNHaswani — COMMENTED** [review]
The contrast in the colors is too sharp. Tone them down a bit

File: pages/cart.html (hunk 1 above)
Comment (darshandamre) [reply pages/cart.html:62]: will change the color theme later, thank you for your time

**Pull request merged**